src/universal/styles/paletteV2.ts

```diff
@@ -16,9 +16,11 @@ export namespace PALETTE {
   }
 
   export const enum BORDER {
-    LIGHT = '#D1CBDB',
+    LIGHTER = '#DFDFE5',
+    LIGHT = '#C1C0CD',
     DARK = '#b6adc7',
-    MAIN = '#493272'
+    MAIN = '#493272',
+    BLUE_RGB = '50, 154, 229'
```

Comment:
can we just bake the alpha into the hex & use that instead? eg `#329AE580`. That way, the alpha is static so i won't be encouraged to play around with the alpha & make slightly wrong colored blues.